docs/docs/writing-your-own-enclave-host.md

```diff
@@ -258,14 +256,13 @@ public static void main(String[] args) throws InvalidEnclaveException, IOExcepti
 }
 ```
 
-Now we need to implement the corresponding logic on the host to receive and process these requests.
+You need to implement the corresponding logic on the host to receive and process the requests.
 
 ## Implementing the host
 
 ### Loading the enclave
 
-One of the first things the host does is [load the enclave](api/-conclave%20-core/com.r3.conclave.host/-enclave-host/load.html), 
-which by default it does by scanning the classpath.
+[Load the enclave](api/-conclave%20-core/com.r3.conclave.host/-enclave-host/load.html) by scanning the classpath.
```

Comment:
"by scanning the classpath" implies the developer has to do this. `Enclave.load()` scans the classpath to find the enclave.